docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc

```diff
@@ -1,84 +1,69 @@
 :imagesdir: ../assets/images
 [[getting-started-migrating-from-opennms-datasource]]
-= Migrating from the OpenNMS Data Source plugin
-:toc: macro
-:toc-title:
-:data-uri:
-:prewrap!:
+= Migrate from the OpenNMS data source plugin
```

Comment:
Which version of Helm was the migration from just a datasource plugin to an app plugin?  Should probably note that here as this doesn't apply to "modern" Helm versions.